README.md

```diff
@@ -47,20 +46,21 @@ Example configuration for CDC features:
-can't be determined it will fail.
+The connector will automatically look up the primary key column for the specified tables. If that can't be determined, 
+the connector will return an error.
 
 ## Configuration Options
 
 | name                      | description                                                                                                                                                                                | required | default       |
 |---------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|---------------|
 | `url`                     | Connection string for the Postgres database.                                                                                                                                               | true     |               |
-| `table`                   | List of table names to read from, separated by comma. Using `*` will read from all public tables.                                                                                          | true     |               |
+| `tables`                  | List of table names to read from, separated by comma. Example: `"employees,offices,payments"`. Using `*` will read from all public tables.                                                 | true     |               |
 | `key`                     | List of Key column names per table, separated by comma. Example:`"table1:key1,table2:key2"`, if not supplied, the table(s) primary keys will be used as the `'Key'` field for the records. | false    |               |
 | `snapshotMode`            | Whether or not the plugin will take a snapshot of the entire table before starting cdc mode (allowed values: `initial` or `never`).                                                        | false    | `initial`     |
 | `cdcMode`                 | Determines the CDC mode (allowed values: `auto`, `logrepl` or `long_polling`).                                                                                                             | false    | `auto`        |
 | `logrepl.publicationName` | Name of the publication to listen for WAL events.                                                                                                                                          | false    | `conduitpub`  |
 | `logrepl.slotName`        | Name of the slot opened for replication events.                                                                                                                                            | false    | `conduitslot` |
+| ~~`table`~~               | List of table names to read from, separated by comma. **Deprecated: use `tables` instead.**                                                                                                | true     |               |
```

Comment:
We have 2 options:

1. Remove the mention of `table` here, since it was removed in the connector.
2. Add `table` back as a supported, but _deprecated_ option in the connector, then we can mention it here as deprecated.

We did the same with `readTime` in the generator (https://github.com/ConduitIO/conduit-connector-generator/pull/73)